frontend/device/pocketbook/device.lua

```diff
@@ -63,6 +86,17 @@ local PocketBook = Generic:new{
     -- Private per-model kludges
     _fb_init = function() end,
     _model_init = function() end,
+    canOpenLink = getLinkOpener,
+    openLink = function(self, link)
+        local enabled, tool = getLinkOpener()
+        logger.info("getlinkOpener: "..tostring(enabled)..", tool:"..tool)
+        logger.info("Link: "..link)
+        logger.info("Link type:"..type(link))
+        if not enabled or not tool or not link or type(link) ~= "string" then return end
```

Comment:
In reply to https://github.com/koreader/koreader/pull/8530/files#r765260999, within this structure of the code, along these lines.
```suggestion
        logger.info("Link: "..link)
        logger.info("Link type:"..type(link))
        if not isCommand(browser_app) or not link or type(link) ~= "string" then return end
```